src/examples/zkapps/voting/membership.ts

```diff
@@ -59,6 +60,13 @@ export class Membership_ extends SmartContract {
 
   reducer = Reducer({ actionType: Member });
 
+  events = {
```

Comment:
just added some dummy events to exercise `fetchEvents` and emit them as soon as the state changes